pages/help/help_account-management.md

```diff
@@ -2,21 +2,66 @@
+- [Get help with your Login.gov account](#get-help-with-your-Login.gov-account)
+- [Access your .gov domain account](#access-your-.gov-domain-account)
+- [Update your contact information (email, phone)](#update-your-contact-information)
+
+## Get help with your Login.gov account
+
+### You must have a Login.gov account to request or manage .gov domains
+
+We use Login.gov to make signing in easier and more secure. If you don’t have a Login.gov account you’ll need to create one to request a .gov domain or to manage existing .gov domains. 
+
+### About Login.gov
+
+Login.gov provides a simple and secure process for signing into many government services with one account. Participating agencies ask you to create a Login.gov account to securely access their websites or applications. Login.gov is managed by the U.S. General Services Administration.
+
+### How to create a Login.gov account
+
+Follow these steps to [create your Login.gov account](https://login.gov/help/get-started/create-your-account/){.usa-link--external}.
+
+### You must verify your identity with Login.gov
+
+We require you to verify your identity with Login.gov. This is an extra layer of security that requires you to prove you are you, and not someone pretending to be you.
+
+After you verify your identity with Login.gov for one government application, you don’t need to do it again for other government applications that use Login.gov and require identity verification. Read more about [verifying your identity with Login.gov](https://login.gov/help/verify-your-identity/how-to-verify-your-identity/){.usa-link--external}. 
 
+When you attempt to access the .gov domain management site with your Login.gov account, we’ll prompt you to verify your identity if you haven’t already done so.
 
+### If you already have a Login.gov account
+
+If you already have a Login.gov account, you can use that account to sign into the .gov domain management site. We’ll prompt you to verify your identity with Login.gov if you haven’t already done so.
+
+### If you already have a .gov account
+
+If you already have a .gov account you’ll need to have a Login.gov account to request or manage your .gov domains.
+
+## Access your .gov domain account
+
+If you already have a .gov account you’ll need to have a Login.gov account to request or manage your .gov domains. Follow these steps to [create your Login.gov account](https://login.gov/help/get-started/create-your-account/){.usa-link--external}.
+
+## Update your contact information (email, phone)
+
+You can change the contact information associated with a .gov domain you manage.
+
+<ol>
+<li>If you don’t already have a Login.gov account, you’ll need to create one to manage existing .gov domains. <a href="https://login.gov/help/get-started/create-your-account/ class="usa-link usa-link--external">Create a Login.gov account</a>.</li> 
+<li>Sign in with your Login.gov account to <a href="#">manage your domain</a>.</li>  
```

Comment:
should it be plural, "manage your domain**s**"? If not should we update the nav and home page buttons to be singular?

Reply:
I wasn't using the phrase here to indicate the proper name of the section. I'm referring to the action. Now that you mention it though...maybe these instructions should start with "Click on 'Manage your domain'..." 🤔